### ISSUE TITLE
Potential ".git"-filtering problem in IoUtil.java

### ISSUE THREAD

**Issue by alexVengrovsk** [codenvy/che-core#201]
Have a closer look at ```GIT_FILTER``` public field:
```
  /** Represent filter what select only .git entries. */
    public static final FilenameFilter GIT_FILTER = new FilenameFilter() {
        @Override
        public boolean accept(File dir, String name) {
            return !(".git".equals(name));
        }
    };
```
According to the comment above the variable it selects only .git entries. But return statement of ```accept``` method returns ```true```, than name is diferent than ".git"

**Comment by alexVengrovsk** [comment]
Maybe comment should be:
```
/** Represent filter, that excludes .git entries. */
```
Otherwise, you should delete "!" sign in return statement.

**Comment by skabashnyuk** [comment]
Fixed in 42bf9c5.
Thank you.
